docs/ml/creating-df-kib.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[creating-df-kib]]
+== Creating {dataframe-transforms}
+
+beta[]
+
+If you want to use the {dataframe-transforms} feature, you must have the required 
+user roles and permissions:
+
+* `data_frame_transforms_user`: to get transforms and transforms stats,
+* `data_frame_transforms_admin`: to create/start/stop/delete transforms,
+* read permissions on the source indices,
+* index permissions on the target index (if the target index does not exist, you 
+need permission to create the index as well).
+
+[role="screenshot"]
+image::ml/images/ml-definepivot.jpg["Defining a {dataframe} pivot"]
+
+You can create {dataframe-transforms} in the {kib} Machine Learning application. 
```

Comment:
Once the security information is removed from the beginning of this page, I recommend using a sentence like this one as the introductory sentence. I also suggest we add a link to https://www.elastic.co/guide/en/elastic-stack-overview/master/ml-dataframes.html in that introduction.